randomizer/text/character_map.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+
+class CharacterMap():
```

Comment:
Hmm...I think I follow what you mean here. Instead of going one by one, we instead take advantage of consecutive characters and use the `ord` function. It's definitely a good alternative, and one I think I can look into.

Reply:
I had to adopt a hybrid approach. `pylint` hated the number of if/else branches and return statements.

Reply:
Nice!  I actually like your hybrid approach better than having one elif statement per special character.